cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1505,7 +1046,8 @@ public Object call(SQLDatabase db) throws Exception {
 
     }
 
-    private String insertNewWinnerRevision(SQLDatabase db, DocumentBody newWinner,
+    // TODO should this be a callable itself?
```

Comment:
I don't think so because it doesn't run against the DB, which I think was the pattern for the callables.